SPECS/dnf/dnf.spec

```diff
@@ -113,7 +118,6 @@ ctest -VV
 %dir %{confdir}/vars
 %exclude %{confdir}/aliases.d/zypper.conf
 %config(noreplace) %{confdir}/%{name}.conf
-%config(noreplace) %{confdir}/protected.d/%{name}.conf
```

Comment:
This concerned me at first since its removing protected.d/dnf.conf, but since we're on libdnf 0.72.0, the dnf package is implicitly considered a protected package https://github.com/rpm-software-management/libdnf/pull/1621/files